Verifying-ERC20.md

```diff
@@ -78,16 +78,14 @@ By executing:  `Verisol ERC20.sol ERC20` we get:
 
 meaning that VeriSol was able to prove the contract complains with the specification.
 
-It is worth noticing that we just  has proven is a *partial* specification since there is an important property missing: the balances of *all* other accounts must remain unchanged!
+It is worth noticing that what we have just  proven is a *partial* specification since there is an important property missing: the balances of *all* other accounts must remain unchanged!
```

Comment:
I would have liked the following specifications instead:

function transfer(address recipient, uint256 amount) public returns (bool) {
  _transfer(msg.sender, recipient, amount);

  assert (msg.sender == recipient || ( _balances[msg.sender] == VeriSol.Old(_balances[msg.sender] - amount));
  assert (msg.sender == recipient || ( _balances[recipient] == VeriSol.Old(_balances[recipient] + amount));
  assert (_balances[msg.sender] <= VeriSol.Old(_balances[msg.sender] );
 assert ( _balances[recipient] >= VeriSol.Old(_balances[recipient]);

To make it symmetric and the last two needed to avoid overflows. 



  return true;

Reply:
I have the full spec with the modifies in ERC20.sol 

I can added to the document.